src/sql/sqlops.proposed.d.ts

```diff
@@ -711,6 +711,13 @@ declare module 'sqlops' {
 				 * done. Return true to allow the dialog to close or false to block it from closing
 				 */
 				registerCloseValidator(validator: () => boolean | Thenable<boolean>): void;
+
+				/**
+				 * Register an operation to run the back ground when the wizard is done
+				 * @param operationInfo Operation Information
+				 * @param handler Operation Handler to run when wizard is done
+				 */
+				registerOperation(operationInfo: BackgroundOperationInfo, handler: (operation: BackgroundOperation) => void): void;
```

Comment:
If OperationInfo is always passed alongside a handler maybe it makes more sense to include the handler inside OperationInfo